contracts/Airdrop/BulkSender.sol

```diff
@@ -0,0 +1,42 @@
+pragma solidity ^0.4.19;
+
+import "@digix/core2/contracts/common/ERCTwenty.sol";
```

Comment:
lets use a ERCTwenty interface that is not from our core2. You can just use the open-zeppelin one or just copy paste from core2. This is like a separate, open source contract tool that anyone could use, with no link to our system.